src/coreclr/jit/importer.cpp

```diff
@@ -9382,6 +9401,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 
                         case CORINFO_FIELD_STATIC_GENERICS_STATIC_HELPER:
                         case CORINFO_FIELD_STATIC_READYTORUN_HELPER:
+                        case CORINFO_FIELD_STATIC_DATASEGMENT:
                             /* We may be able to inline the field accessors in specific instantiations of generic
```

Comment:
This block exists because of inlining into shared generic code. Why does `CORINFO_FIELD_STATIC_DATASEGMENT` need to be there?

Reply:
Ah, right, copy-paste. In this case we're not going to emit a helper (we probably still might due to type initiazation?) but it should be fine.